autoload/neomake.vim

```diff
@@ -42,6 +42,13 @@ function! neomake#GetJob(job_id) abort
     return s:jobs[a:job_id]
 endfunction
 
+function! neomake#GetErrors(bufnr) abort
+    if !a:bufnr
+        return s:current_errors['project']
+    endif
+    return get(s:current_errors['file'], a:bufnr, {})
+endfunction
+
```

Comment:
Yes, and the statusline API in general.
My plan is to clean up my local config some day for that..